practice-two/src/index.html

```diff
@@ -12,11 +12,11 @@
   <!-- sign up start  -->
   <div class="sign-up">
     <div class="container sign-up">
```

Comment:
The parent of this element has class sign-up. Why you still using this class here?